ci-operator/step-registry/baremetalds/devscripts/proxy/baremetalds-devscripts-proxy-commands.sh

```diff
@@ -37,12 +37,19 @@ sudo systemctl start firewalld
 sudo firewall-cmd --add-port=8213/tcp --permanent
 sudo firewall-cmd --reload
 
+
+SQUID_IMAGE=quay.io/sameersbn/squid:latest
+if [ "${ARCHITECTURE}" == "arm64" ]; then
+  # Test image reference, will update at a later date
+  SQUID_IMAGE=quay.io/openshifttest/squid-proxy:multiarch
+fi
```

Comment:
why not just use the `openshifttest` image on all arches?

Reply:
We would have to test `openshifttest` image

Reply:
Testing now.

Reply:
I updated this section to include the multiarch image, and it seems to not cause any problems.  Will resolve the thread here.